src/unframed/index.js

```diff
@@ -2,6 +2,12 @@ import { D2LFetchAuth } from './d2lfetch-auth.js';
 
 const fetchAuth = new D2LFetchAuth();
 
-module.exports = function auth(request, next) {
+module.exports = function auth(request, next, options) {
+	if (options) {
+		fetchAuth._enableTokenCache = (options.enableTokenCache === true);
+		if (options._resetLocalCache) {
```

Comment:
Por que the `_`, while `enableTokenCache` doesn't have one?

Reply:
> Needed to update the tests a bit and add an option the tests can pass in to invalidate the local cache.

Reply:
Yeah it's not clear, but this was meant to be an "internal option" that nobody should know about. Shhhh, it's secret.